externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/ui/EnginePage.scala

```diff
@@ -365,8 +366,19 @@ private class StatementStatsPagedTable(
         {event.state}
       </td>
       <td>
-        {event.queryExecution}
+        {
```

Comment:
Actually, I don't like this code style.

Reply:
yeah, it seems not aligned with line 379 ?

Reply:
cc @pan3793 @zhouyifan279